docker/Dockerfile

```diff
@@ -59,9 +60,6 @@ RUN git config --global http.sslverify false && \
 ## Setup environment for Serialbox
 ##---------------------------------------------------------------------------------
 FROM fv3gfs-environment as fv3gfs-environment-serialbox
-# set TZ
-ENV DEBIAN_FRONTEND=noninteractive TZ=US/Pacific
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
This is now handled [as it is in the fv3net prognostic_run image](https://github.com/ai2cm/fv3net/blob/3b1ff5c9378f6a70c8b760e4f02df373d32de780/.environment-scripts/gnu_docker/install_base_software.sh#L12).